Skaffold panics if Dockerfile contains 'COPY' line

<!--
  Issues without logs and details are more complicated to fix.
  Please help us by filling the template below!
-->
Skaffold panics when a dockerfile it parses contains a line with "COPY" with no args. Such dockerfile is incorrect, but it should not lead to crash.

### Expected behavior
A well-formatted error message describing the dockerfile issue
### Actual behavior
```shell
izuyev@izuyev-macbookpro --- examples/getting-started ‹add_perf_tooling* M?› » skaffold run
Generating tags...
 - skaffold-example -> skaffold-example:v1.14.0-3-g5b1f94025-dirty
Checking cache...
panic: runtime error: slice bounds out of range [:-1]

goroutine 13 [running]:
github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker.readCopyCommand(0xc0007b0700, 0xc000859680, 0x0, 0x0, 0xc0007cfc80, 0x3, 0xc0004ff9b0, 0xc0008340c0, 0xc0001800e0)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/docker/parse.go:278 +0x8c6
github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker.extractCopyCommands(0xc0008340c0, 0x8, 0xc, 0xc0004fe800, 0xc0004fe8d0, 0x8, 0xc, 0x0, 0x0, 0xc000a70730)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/docker/parse.go:239 +0x3d7
github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker.readCopyCmdsFromDockerfile(0x0, 0xc000a70730, 0x43, 0x5a3dc55, 0x1, 0x0, 0xc0004fe8d0, 0x0, 0x0, 0x0, ...)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/docker/parse.go:90 +0x37a
github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker.GetDependencies(0x5d7e820, 0xc000531140, 0x5a3dc55, 0x1, 0x5a46d27, 0xa, 0x0, 0xc0004fe8d0, 0xc00082f430, 0x10, ...)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/docker/dependencies.go:61 +0x25e
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build.DependenciesForArtifact(0x5d7e820, 0xc000531140, 0xc0003ea660, 0xc0004fe8d0, 0x5462d39, 0xc00084fbf0, 0x404ef05, 0xc00082f430, 0x0)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/build/dependencies.go:42 +0x47c
github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner.NewForConfig.func1(0x5d7e820, 0xc000531140, 0xc0003ea660, 0x0, 0x1, 0xc00082f430, 0x0, 0x1)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/runner/new.go:69 +0x76
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache.getHashForArtifact(0x5d7e820, 0xc000531140, 0xc00078a180, 0xc0003ea660, 0x5a3e59c, 0x3, 0x0, 0x0, 0x0, 0x0)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/build/cache/hash.go:56 +0x20f
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache.NewCache.func1(0x5d7e820, 0xc000531140, 0xc0003ea660, 0x0, 0x0, 0x0, 0x0)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/build/cache/cache.go:100 +0x7c
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache.(*cache).lookup(0xc0003ea3c0, 0x5d7e820, 0xc000531140, 0xc0003ea660, 0xc0000580c0, 0x2b, 0x0, 0x0)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/build/cache/lookup.go:51 +0x66
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache.(*cache).lookupArtifacts.func1(0xc0003ea3c0, 0x5d7e820, 0xc000531140, 0xc00047f300, 0x1, 0x4, 0x0, 0xc0004feb10, 0xc00082f3d0, 0x1, ...)
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/build/cache/lookup.go:41 +0xb5
created by github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/cache.(*cache).lookupArtifacts
        /Users/izuyev/work/sk_z/skaffold/pkg/skaffold/build/cache/lookup.go:40 +0x134
```
### Information
```shell
izuyev@izuyev-macbookpro --- examples/getting-started ‹add_perf_tooling* M?› » skaffold version                                                                                                            2 ↵
v1.14.0-24-gf3842fa91-dirty
```
I'm using the "skaffold/examples/getting-started" project with modified Dockerfile
```shell
izuyev@izuyev-macbookpro --- examples/getting-started ‹add_perf_tooling* M?› » cat skaffold.yaml 
apiVersion: skaffold/v2beta7
kind: Config
build:
  artifacts:
  - image: skaffold-example
deploy:
  kubectl:
    manifests:
      - k8s-*
```

``` shell
izuyev@izuyev-macbookpro --- examples/getting-started ‹add_perf_tooling* M?› » cat Dockerfile 
FROM golang:1.12.9-alpine3.10 as builder
COPY #  <== The problem line
COPY main.go .
RUN go build -o /app main.go

FROM alpine:3.10
# Define GOTRACEBACK to mark this container as using the Go language runtime
# for `skaffold debug` (https://skaffold.dev/docs/workflows/debug/).
ENV GOTRACEBACK=single
CMD ["./app"]
COPY --from=builder /app .
```

--- a/pkg/skaffold/docker/parse.go
+++ b/pkg/skaffold/docker/parse.go
@@ -272,6 +272,9 @@ func readCopyCommand(value *parser.Node, envs []string, workdir string) (*copyCo
 
 		paths = append(paths, path)
 	}
+	if len(paths) == 0 {
+		return nil, fmt.Errorf("invalid dockerfile instruction: %q", value.Original)
+	}
 
 	// All paths are sources except the last one
 	var srcs []string

